src/spawn.cpp

```diff
@@ -100,7 +100,8 @@ bool Spawns::loadFromXml(const std::string& filename)
 					centerPos.y + pugi::cast<uint16_t>(childNode.attribute("y").value()),
 					centerPos.z
 				);
-				int32_t interval = pugi::cast<int32_t>(childNode.attribute("spawntime").value()) * 1000;
+
+				int32_t interval = pugi::cast<int32_t>(childNode.attribute("spawntime").value()) * 1000 * (g_config.getNumber(ConfigManager::SPAWN_INTERVAL_RATE) / 100);
 				if (interval >= MINSPAWN_INTERVAL && interval <= MAXSPAWN_INTERVAL) {
```

Comment:
There is a min and a max, should we document those in config.lua to make user aware?